cfme/tests/cloud_infra_common/test_retirement.py

```diff
@@ -33,6 +34,33 @@ def _delete_vm():
     return vm_obj
 
 
+@pytest.fixture(scope="function")
+def existing_vm(request):
+    list_of_existing_providers = providers.existing_providers()
+    if list_of_existing_providers:
+        test_provider = providers.get_crud(list_of_existing_providers[0])
+    else:
+        test_provider = providers.setup_provider("vsphere55")
+    all_vms = test_provider.mgmt.list_vm()
+    need_to_create_vm = True
+    for virtual_machine in all_vms:
+        if test_provider.mgmt.is_vm_running(virtual_machine):
+            need_vm = VM.factory(virtual_machine, test_provider)
+            need_to_create_vm = False
+            break
+    if need_to_create_vm:
+        machine_name = 'test_retire_prov_{}'.format(fauxfactory.gen_alpha(length=8).lower())
+        deploy_template(test_provider.key,
```

Comment:
You don't need `deploy_template` here, you can use the `vm_obj.create_on_provider` that you can see being used in the `vm` fixture above.